### ISSUE TITLE
bug: absolute import used in flow cannot be used for autocomplete

### ISSUE THREAD

**Issue by ImSingee** [windmill-labs/windmill#3937]
### Describe the bug

When using `import {} from "/f/xx/xx.ts"`, the autocomplete features won't work.

### To reproduce

1. Create a script `/f/test/test.ts`
2. Create a flow, create a new inline bun script, write `import {} from "/f/test/test.ts"`
3. The editor will show `Cannot find module '/f/test/test.ts' or its corresponding type declarations.(2307)`

### Expected behavior

The editor shouldn't report any errors

### Screenshots

<img width="1017" alt="image" src="https://github.com/windmill-labs/windmill/assets/11208082/57239e57-4dcc-4819-a364-7302b7767e5b">

### Browser information

Arc Version 1.47.1 (50893) Chromium Engine Version 126.0.6478.57

### Application version

Cloud Version (EE v1.349.0-3-ga82184beb)

### Additional Context

In the console, the editor downloads the script successfully:
<img width="1437" alt="image" src="https://github.com/windmill-labs/windmill/assets/11208082/0f345c59-7aa7-428b-9e57-df352d5acf6b">

The bug is only for the editor. The script can be executed successfully.

The bug is only for flow; it doesn't exist in the script.

I create a dedicated windmill cloud workspace for this bug report, you can view and test it directly in `bryan-test` workspace.

**Comment by ImSingee** [comment]
And same for deno

<img width="991" alt="image" src="https://github.com/windmill-labs/windmill/assets/11208082/e2e682b1-0bc1-4bad-a1c7-9505b023c3f8">

**Comment by ImSingee** [comment]
And everything works well when using relative imports:
<img width="606" alt="image" src="https://github.com/windmill-labs/windmill/assets/11208082/a0d3191d-4f39-4469-bcd3-0971ea744196">

But not if using relative imports in branch:
<img width="1398" alt="image" src="https://github.com/windmill-labs/windmill/assets/11208082/e351bea9-8f37-4c02-b42d-0d93879caa3b">

**Comment by ImSingee** [comment]
This is the exported file for my test workspace:

[windmill-bryan-test.zip](https://github.com/user-attachments/files/15909636/windmill-bryan-test.zip)

**Comment by whollacsek** [comment]
Yeah import path typing needs to be fixed, just spent hours trying different workaround but none is working so I'm just going to stick to absolute path for now

**Comment by rubenfiszel** [comment]
We likely cannot make work the absolute imports in the editor but all the other issues should be fixed on latest. Thanks for the report. I strongly recommend using only relative imports, absolute imports are too hacky imho, and should only be used as an escape hatch (and the issues here being solved, that's one less reason to not use them)
